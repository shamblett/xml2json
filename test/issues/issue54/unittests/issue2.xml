<?xml version="1.0" encoding="utf-8"?>
<!--
 Test Case submitted in issue 2: Invalid octals
 http://code.google.com/p/xml2json-xslt/issues/detail?id=2
 -->
<root>
  <octal>01234567</octal>
  <notoctal>089</notoctal>
</root>
