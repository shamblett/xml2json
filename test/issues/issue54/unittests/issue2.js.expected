/*
 Test Case submitted in issue 2: Invalid octals
 http://code.google.com/p/xml2json-xslt/issues/detail?id=2
 */{
    octal:"01234567",
    notoctal:"089"}
